pages/content/amp-dev/about/_blueprint.yaml

```diff
@@ -1,4 +1,4 @@
-$title@: About
+$title@: What is AMP?
```

Comment:
I think Longos mock was outdated here, we agreed with the AMP folks to really use "About". Sorry.

Reply:
sorry, forgot to implement this one @matthiasrohmer , doing it now